examples/carla/Carla_Challenge/carlaChallenge10.scenic

```diff
@@ -43,16 +45,16 @@ adv_trajectory = [adv_maneuver.startLane, adv_maneuver.connectingLane, adv_maneu
 adv_start_lane = adv_maneuver.startLane
 
 ## OBJECT PLACEMENT
-# Use the -1' index to get the last endpoint from the list of centerpoints in 'centerline'
-ego_spawn_point = ego_start_lane.centerline[-1]
-adv_spawn_point = adv_start_lane.centerline[-1]
+ego_spawn_pt = OrientedPoint in ego_maneuver.startLane.centerline
+adv_spawn_pt = OrientedPoint in adv_maneuver.startLane.centerline
 
-# Set a specific vehicle model for the Truck.
-# The referenceable types of vehicles supported in carla are listed in scenic/simulators/carla/model.scenic
-# For each vehicle type, the supported models are listed in scenic/simulators/carla/blueprints.scenic
-ego = Car following roadDirection from ego_spawn_point for EGO_DISTANCE_TO_INTERSECTION,
-	with behavior EgoBehavior(ego_trajectory)
+ego = Car at ego_spawn_pt,
+    with blueprint EGO_MODEL,
+    with behavior EgoBehavior(EGO_SPEED, ego_trajectory)
 
-adversary = Car following roadDirection from adv_spawn_point for ADV_DISTANCE_TO_INTERSECTION,
-	with behavior CrossingCarBehavior(adv_trajectory)
+adversary = Car at adv_spawn_pt,
+    with behavior AdversaryBehavior(adv_trajectory)
 
+require (distance to intersec) in Range(20, 25)
```

Comment:
This only works due to what is arguably a bug in Scenic - the `Range` is a distribution and should get sampled to a single number, at which point the `in` operator wouldn't work. In case I fix this bug later, it would be safer to simply write `20 <= (distance to intersec) <= 25`.

Reply:
@dfremont Thanks for the quick review! Updated scenarios to avoid using `Range` in those situations.